release/distribution/NOTICE

```diff
@@ -0,0 +1,36 @@
+The following copyright statements and licenses apply to various third party open
+source software packages (or portions thereof) that are distributed with
+this content.
+
+TABLE OF CONTENTS
+=================
+
+The following is a listing of the open source components detailed in this
+document.  This list is provided for your convenience; please read further if
+you wish to review the copyright notice(s) and the full text of the license
+associated with each component.
+
+
+**SECTION 1: Apache License, V2.0**
+  * Develocity Maven Extension Adapters
+    * com.gradle:develocity-maven-extension-adapters
```

Comment:
Something Maven related in the Gradle plugin?

Reply:
Great catch. It should be `com.gradle:develocity-gradle-plugin-adapters`. I must have only noticed `adapters` and overlooked the build tool. 

https://ge.solutions-team.gradle.com/s/rh6xon4ychuli/dependencies?focusedDependency=WzAsMiwxLFswLDIsWzFdXV0&toggled=W1swXSxbMCwyXV0

Reply:
Got it. I assume you will adjust all occurrences. No need for me to re-review.

Reply:
Fixed – but I want to wait to merge until I can verify all JARs include the expected files when published to Maven local, however [I can't right now](https://gradle.slack.com/archives/C04U1H825D5/p1725375229470849).

Reply:
Fixed – but I want to wait to merge until I can verify all JARs include the expected files when published to Maven local, however I can't right now because the publish task fails. This is being discussed via Slack internally.